pkg/database/migrations.go

```diff
@@ -2359,6 +2359,26 @@ func (db *Database) Migrations(ctx context.Context) []*gormigrate.Migration {
 				)
 			},
 		},
+		{
+			ID: "00109-MakeWebViewARealmSetting",
+			Migrate: func(tx *gorm.DB) error {
+				return multiExec(tx,
+					`ALTER TABLE realms
+						ADD COLUMN IF NOT EXISTS allow_user_report_web_view BOOLEAN`,
+					`ALTER TABLE realms
+						ALTER COLUMN allow_user_report_web_view SET DEFAULT false`,
+					`ALTER TABLE realms
+						ALTER COLUMN allow_user_report_web_view SET NOT NULL`,
+					// This was missing in a previous migration.
+					`ALTER TABLE realms
```

Comment:
Can you make this 109 by itself and then add 110 with the allow_user_report_web_view (just in case we need to rollback)